WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -3481,9 +3484,9 @@ function WeakAuras.PositionOptions(id, data, hideWidthHeight, disableSelfPoint)
         end
         return not data.anchorPoint:find("LEFT")
       end,
-      get = function() return data.xOffset end,
+      get = function() return data.xOffset*GetScale() end,
       set = function(info, v)
-        data.xOffset = v;
+        data.xOffset = v*(1/GetScale());
```

Comment:
I'm curious why you did this? I would have expected the unscaled values here.

Reply:
Problem here was that as you scale something, their coordinate system change. So 100px offset at 0.8 scale would look like 80px. So if you want align 2 groups (1 scaled and other isnt) you wouldn't be able to just use same offsets (assuming same anchor point). 
Also movers used scaled values so a bit confusing there when mover is showing one result, while options another.
Basically just to keep using same coordinate system for user.

Reply:
But the sizes of the auras use the changed coordinate system, right? So aligning in a group is now harder? That feels like the wrong solution.

The mover can be fixed.